src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -2144,13 +2144,9 @@ - (NSArray *)_getAttributesToReportWithReportedValues:(NSArray<NSDictionary<NSSt
             // if expected values exists, purge and update read cache
```

Comment:
This comment is now completely wrong.   There should be some comments here that actually explain what is going on, though, and those are very much missing.

Reply:
Filed https://github.com/project-chip/connectedhomeip/issues/33013